app/src/main/java/com/example/marvelapp/domain/entity/SeriesEntity.kt

```diff
@@ -1,7 +1,10 @@
 package com.example.marvelapp.domain.entity
 
+import com.example.marvelapp.data.local.model.utils.ConstantsUtils.EMPTY_STRING
+import com.example.marvelapp.data.local.model.utils.ConstantsUtils.VALUE_ZERO
+
 class SeriesEntity(
-    val available: Int,
-    val collectionURI: String,
-    val returned: Int
+    val available: Int = VALUE_ZERO,
```

Comment:
these default values VALUE_ZERO and EMPTY_STRING should be in the domain package. You should'n use the constants from data. Create a new ConstantsUtils here in domain

Reply:
check this for all the domain entities

Reply:
Thx, now its done